src/routes/+page.svelte

```diff
@@ -519,72 +520,37 @@
-                                <span class="hidden md:flex">{anime.current_episode}/{anime.episodes_count}</span>
-                                <span class="md:hidden">Episode {anime.current_episode}/{anime.episodes_count}</span>
-                            </episode-details>
-                        </anime-details>
-
-                        <hover-buttons class="absolute inset-0 flex flex-col items-center justify-center gap-2 opacity-0 transition-opacity duration-300 group-hover:opacity-100 md:gap-[0.5vw]">
-                            <button class="btn btn-icon h-14 w-24 rounded-xl bg-primary-500 text-base font-bold md:h-[3.125vw] md:w-[5.4375vw] md:rounded-[0.625vw] md:text-[0.875vw]">
-                                <PlayCircle class="w-4 md:w-[1vw]" />
-                                <span>Ep {anime.current_episode}</span>
-                            </button>
-
-                            <button class="btn btn-icon h-14 w-24 rounded-xl bg-surface-900 text-base font-bold text-surface-50 md:h-[3.125vw] md:w-[5.4375vw] md:rounded-[0.625vw] md:text-[0.875vw]">
-                                <Info class="w-5 md:w-[1.25vw]" />
-                                <span>Info</span>
-                            </button>
-                        </hover-buttons>
-                    </card>
+                        <overlay class="absolute inset-0 flex items-end md:p-[0.5vw] leading-none bg-gradient-to-t from-surface-900/75 to-transparent">
+                            <div class="md:rounded-[0.3vw] overflow-hidden flex md:gap-[0.2vw]">
+                                <subs class="bg-warning-400 text-black md:px-[0.35vw] md:py-[0.25vw] flex items-center md:gap-[0.25vw]">
+                                    <Caption class="md:h-[1.25vw]" />
+                                    <span class="md:text-[0.8vw] font-semibold">{anime.episodes_count}</span>
+                                </subs>
+                                <dubs class="bg-white/25 backdrop-blur md:px-[0.45vw] md:py-[0.25vw] flex items-center md:gap-[0.25vw]">
+                                    <Mic class="md:h-[0.8vw]" />
+                                    <span class="md:text-[0.8vw] font-semibold">{anime.episodes_count}</span>
+                                </dubs>
+                            </div>
+                        </overlay>
+                    </anime-cover>
+
+                    <anime-details class="flex flex-col md:gap-[0.5vw]">
+                        <anime_name class="line-clamp-1 text-base font-semibold md:text-[1.1vw] leading-none">{anime.name}</anime_name>
+                        <anime_info class="flex items-center gap-2 text-sm text-surface-50 md:gap-[0.5vw] md:text-[0.9vw] leading-none">
+                            <span>Watching</span>
+                            <Circle class="w-1 md:w-[0.25vw]" />
```

Comment:
Why use `Circle`?

Reply:
Why not?
`after:content` this is hard to maintain

Reply:
I still like `after:content`

Reply:
> I still like after:content

I cant go with `after::`
wanna use `RoundSquare` instead?

Reply:
No use `<Tattoo />`